op-chain-ops/cmd/check-l2/main.go

```diff
@@ -847,6 +940,17 @@ func checkEAS(addr common.Address, client *ethclient.Client) error {
 	return nil
 }
 
+func checkPredeployBytecode(addr common.Address, name string, client *ethclient.Client, expectedBytecode []byte) error {
```

Comment:
This function doesn't necessarily need the name, error wrapping from the caller could decorate the error with `fmt.Errorf` including the name. This does dedupe a bunch of common `fmt.Errorf` wrapping so it might be less annoying